pages/index.jsx

```diff
@@ -36,3 +41,16 @@ const IssuesPage = () => {
 };
 
 export default IssuesPage;
+
+export const getServerSideProps: GetServerSideProps<{
```

Comment:
its gives a syntax error on data part in the `getServerSideProps` method. but I followed Next.js docs on it.

Reply:
changed to getStaticProps after research